firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -314,3 +315,64 @@ uint8_t TCMD_agenda_fetch(){
 
     return 0;
 }
+
+
+/// @brief Deletes a telecommand from the agenda by the telecommand function name.
```

Comment:
What if there are multiple telecommands with the same name?

Reply:
It removes all instances with the name that's passed into the function from the agenda. Are we trying to implement this differently?

Reply:
I updated the docstring to match the functionality of the function